rtt/typekit/BoolTypeInfo.hpp

```diff
@@ -49,10 +49,10 @@ namespace RTT
         /**
          * Write boolean as 'true' or 'false'.
          */
-        struct BoolTypeInfo: public TemplateTypeInfo<bool,true>
+        struct BoolTypeInfo: public StdTypeInfo<bool>
         {
             BoolTypeInfo() :
-                TemplateTypeInfo<bool,true> ("bool")
+                StdTypeInfo<bool> ("bool")
             {
```

Comment:
AFAIK this change is NOT ABI compatible. See [1]. You are modifiying the signature of a struct.

[1] https://techbase.kde.org/Policies/Binary_Compatibility_Issues_With_C++